channel.go

```diff
@@ -485,6 +485,43 @@ func (ch *Channel) RejectInvite(userID string, message *Message) error {
 	return ch.client.makeRequest(http.MethodPost, p, nil, data, nil)
 }
 
+func (ch *Channel) Mute(userID string, expiration *time.Duration) (*ChannelMuteResponse, error) {
+	if userID == "" {
+		return nil, errors.New("user ID must be not empty")
+	}
+
+	if expiration != nil {
+		*expiration /= time.Millisecond
```

Comment:
This is technically correct, but we shouldn't be doing it. We should be creating a new non-`time.Duration` variable and use that instead of converting a `time.Duration` in place and making it no longer a real `time.Duration`. Also `int(expiration.Seconds())` should do what is needed here.